Resfeber/View Controllers/TripsViewController.swift

```diff
@@ -10,18 +10,21 @@ import UIKit
 
 class TripsViewController: UIViewController {
     // MARK: - Properties
-    
+
     fileprivate let destinationController = DestinationController()
     fileprivate var collectionView: UICollectionView!
 
+    fileprivate var tripService: TripService! = nil
```

Comment:
Any reason why `tripService` is being set in `viewDidLoad()` instead of just setting its initial value directly? If you set its value here then it does not have to be an explicitly unwrapped optional. I think it can also be changed from a `var` to a `let` if you initialize it here.
```suggestion
    fileprivate let tripService = TripService()
```